site2/docs/security-authorization.md

```diff
@@ -98,3 +98,15 @@ PulsarAdmin admin = PulsarAdmin.builder()
                     .tlsTrustCertsFilePath("/path/to/trust/cert")
                     .build();
 ```
+
+## Authorize an authenticated client with multiple roles
+
+When a client is identified with multiple roles in a token (the type of role claim in the token is an array) during the authentication process, Pulsar supports to check the permissions of all the roles and further authorize the client as long as one of its roles has the required permissions.
+
+> Note: This authorization method is only compatible with [JWT authentication](security-jwt.md).
+
+To enable the support, configure the authorization provider as `MultiRolesTokenAuthorizationProvider` in the `conf/broker.conf` file.
```

Comment:
do we need to configure it in the `conf/standalone.conf` file? @RobertIndie

Reply:
We only need to do that when we use the standalone mode. But I think it's no need to mention it here.